bridge/cypress/support/pageobjects/TriggerSequenceSubPage.ts

```diff
@@ -85,7 +85,16 @@ export class TriggerSequenceSubPage {
     return this;
   }
 
-  public selectDateTime(calElement: number, hours: string, minutes: string, seconds: string): this {
+  public selectDateTime(
+    calElement: number,
+    hours: string,
+    minutes: string,
+    seconds: string,
+    goBackMonths: number
+  ): this {
+    for (let i = 0; i < goBackMonths; ++i) {
+      cy.get('.dt-calendar-header-button-prev-month').click();
+    }
```

Comment:
Suggestion: You could do
```
[...Array(goBackMonths).keys()].forEach(() => cy.get('.dt-calendar-header-button-prev-month').click())
```
To get rid of for loop

Reply:
This is still a for-loop, even a more complicated one.
You create an array with the length of 5, get the keys via iterator, create another array out of it and then iterate over it.
Also `array.forEach()` is as the same as `for (const i of array)`

Reply:
I completely disagree here and for me, the for-loop approach is much more read-friendly than the forEach